recipe/meta.yaml

```diff
@@ -61,5 +61,9 @@ about:
 
 extra:
   recipe-maintainers:
+    - dillon-cullinan
```

Comment:
@dillon-cullinan, I've added you as a maintainer here. Please let me know if that is ok. 🙂

Reply:
Dillon, if you decide you don't want to be listed here, please just let us know and we can remove you. Thanks!